option optimize_for = LITE_RUNTIME;

message MonthCardGetRequest
{
 required uint32 id = 1;
}
